bundle/compliance/cis_aws/rules/cis_1_20/test.rego

```diff
@@ -0,0 +1,57 @@
+package compliance.cis_aws.rules.cis_1_20
+
+import data.compliance.cis_aws.data_adapter
+import data.lib.test
+
+generate_input(analyzers) = {
+	"resource": {
+		"Analyzers": analyzers,
+		"Region": "eu-west-1",
+	},
+	"subType": "aws-region-accessanalyzers",
+	"type": "identity-management",
+}
+
+test_violation {
+	eval_fail with input as generate_input(null)
+	eval_fail with input as generate_input([])
+	eval_fail with input as generate_input([{"Arn": "some-arn"}])
+	eval_fail with input as generate_input([{
+		"Arn": "some-arn",
+		"Status": "FOO",
+	}])
+}
+
+test_pass {
+	eval_pass with input as generate_input([{
+		"Arn": "some-arn",
+		"Status": "ACTIVE",
+	}])
+	eval_pass with input as generate_input([
```

Comment:
Seems that this one should fail, as the rule said it should be enabled in all regions, no?

Reply:
Each item in the array is an analyzer for one region. The rule is applied once per region.

Reply:
Thank you for the explanation. In that case I think we need to update the rule as it says `Ensure that IAM Access analyzer is enabled for all regions`. So we should fail in case that the analyzer is not enabled at least in one region, no? wdyt?

Reply:
Updated